.taskcluster.yml

```
@@ -0,0 +1,26 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $match:
+    '(tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"])':
+      taskId:
+        $eval: as_slugid("pr_task")
+      provisionerId: proj-getting-started
+      workerType: tutorial
+      payload:
+        maxRunTime: 3600
+        image: node
+        command:
+          - /bin/bash
+          - '--login'
+          - '-c'
+          - >-
+            git clone {{event.head.repo.url}} repo && cd repo && git config
+            advice.detachedHead false && git checkout {{event.head.sha}} && npm
+            install . && npm test
```

```suggestion
            install . && npm lint && npm test
```

maybe?

---

Oh, that's funny, I thought I did that change, but I don't see it here. I guess I forgot to update my commit before pushing.

---

That's the correct syntax though:

```suggestion
            install . && npm run lint && npm test
```